server/services/licenceService.ts

```diff
@@ -198,6 +201,39 @@ export default class LicenceService {
     return this.licenceApiClient.updateAdditionalConditionData(licenceId, additionalConditionId, requestBody, user)
   }
 
+  async uploadExclusionZoneFile(
+    licenceId: string,
+    additionalConditionId: string,
+    fileToUpload: Express.Multer.File,
+    user: User,
+    testMode = false
```

Comment:
Whats this for?

Reply:
In some service tests, it doesn't actually create the file as the `licenceApiClient` is mocked. The test mode prevents it trying to remove this file.